src/main/features/response/components/payment-intention/payment-option.ts

```diff
@@ -0,0 +1,92 @@
+import * as express from 'express'
+
+import { Paths, FullAdmissionPaths } from 'response/paths'
+import { Paths as PaymentIntentionPaths } from 'response/components/payment-intention/paths'
+
+import { ErrorHandling } from 'main/common/errorHandling'
+import { DefendantPaymentOption, DefendantPaymentType } from 'response/form/models/defendantPaymentOption'
+import { Form } from 'main/app/forms/form'
+import { FormValidator } from 'main/app/forms/validation/formValidator'
+import { User } from 'main/app/idam/user'
+import { DraftService } from 'services/draftService'
+import { FeatureToggleGuard } from 'main/app/guards/featureToggleGuard'
+import { ResponseDraft } from 'response/draft/responseDraft'
+import { Draft } from '@hmcts/draft-store-client'
+import { Claim } from 'main/app/claims/models/claim'
+import { FeatureToggles } from 'utils/featureToggles'
+
+export class PaymentOptionPage {
```

Comment:
this code seems specific to admissions does it need to be / should it be?

Reply:
Part which is not welcomed there is SoM as payment intention component should not be aware of this. There is definitely room for improvement.

Reply:
Guards potentially could have been moved outside as well.